spam/03_spam_data_slicing_tutorial.py

```diff
@@ -438,8 +444,10 @@ def textblob_polarity(x):
 # ### Representation learning with slices
 
 # %% [markdown]
-# To cope with scale, we will attempt to learn and combine many slice-specific representations with an attention mechanism (for more, please see our technical report — coming soon!).
-# Using the helper, `convert_to_slice_tasks`, we have now have a list of slice tasks that appropriate constructs the `task_flow` to do just that!
+# To cope with scale, we will attempt to learn and combine many slice-specific representations with an attention mechanism.
+# (For details, please see our technical report — coming soon!)
```

Comment:
Can we set reminder(s) somewhere to update this when that's available?

Reply:
done